src/core/three/Glitch.vue

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+import { GlitchPass } from 'three/examples/jsm/postprocessing/GlitchPass.js'
+import { makePropWatchers } from '../../util/prop'
+import { useEffect } from './composables/useEffect'
+
+export interface GlitchProps {
+  /**
+   * The size of the generated noise map
+   */
+  dtSize?: number
+  /**
+   * If true, the glitch effect will be more wild 🤪
+   */
+  goWild?: boolean
+}
+</script>
+
+<script lang="ts" setup>
+const props = defineProps<GlitchProps>()
+
+const { pass } = useEffect(() => new GlitchPass(props.dtSize))
```

Comment:
@Tinoooo does this pass have a callback every time they trigger the glitch? Since it could be considered as an animation we should invalidate the frames otherwise it will only render the effect once while using `on-demand` (see image below)

If the effect has no callback, maybe we can fake it by only adding an `onBeforeRender` that invalidates once this is mounted

![Screenshot 2024-09-11 at 10 16 17](https://github.com/user-attachments/assets/fb5964d6-7b22-4d82-acef-e56aca74098f)

Reply:
I added it for glitch and noise. For some reason the outline effect does not work with on-demand rendering. Also on main. I added a bug issue for that (https://github.com/Tresjs/post-processing/issues/125).